# Goal specifies a group name
string group_name
uint8 TYPE_UNKNOWN=0
uint8 TYPE_SINGLE_HANDED=1
uint8 TYPE_NONVERBAL_LOW=2
uint8 TYPE_NONVERBAL_HIGH=3
uint8 type
---
# Result
bool success
---
# Feedback
uint8 PHASE_UNKNOWN=0
uint8 PHASE_APPROACH=1
uint8 PHASE_WAITING_FOR_CONTACT=2
uint8 PHASE_EXECUTING=3 #shaking, opening, closing
uint8 PHASE_RETREAT=4
uint8 phase
